apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/TransformerTestUtils.java

```diff
@@ -0,0 +1,29 @@
+package gov.cms.bfd.pipeline.rda.grpc.source;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.function.ToIntFunction;
+import java.util.stream.Collectors;
+
+public class TransformerTestUtils {
+  /**
+   * The McsClaimTransformerTest and FissClaimTransformerTest need to be able to compare lists of
+   * objects and there isn't a standard matcher that is compatible and handles ordering properly.
+   */
+  public static <T> void assertListContentsHaveSamePropertyValues(
```

Comment:
Does `assertEquals(set1, set2)` not work here?  Do the entities have `@EqualsAndHash` defined?

Reply:
Sadly we need to use the reflection based property matching rather than plain equality.  The entities follow the hibernate recommended practice of only including primary key fields in the equals/hashcode methods.